From 64ec7c26404f4a315d20ef0971ba521976afcccd Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Tue, 14 Mar 2017 20:53:09 +0100
Subject: quartz: fix build for <10.12

The style mask type was changed from NSUInteger to NSWindowStyleMask.
Use NSUInteger so it also works on older macOS.

https://bugzilla.gnome.org/show_bug.cgi?id=780019
---
 gdk/quartz/gdkwindow-quartz.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdk/quartz/gdkwindow-quartz.c b/gdk/quartz/gdkwindow-quartz.c
index f868f35..6878e52 100644
--- gdk/quartz/gdkwindow-quartz.c
+++ gdk/quartz/gdkwindow-quartz.c
@@ -2496,7 +2496,7 @@ gdk_quartz_window_set_functions (GdkWindow    *window,
 
   if (impl->toplevel)
     {
-      NSWindowStyleMask mask = [impl->toplevel styleMask];
+      NSUInteger mask = [impl->toplevel styleMask];
 
       if (min)
         mask = mask | NSMiniaturizableWindowMask;
-- 
cgit v0.12
